### ISSUE TITLE
I found that the configuration of test_net_config.conf is not the configuration node of shasta.

### ISSUE THREAD

**Issue by apa7** [tronprotocol/java-tron#2660]
### 1. What did you do?

config:
seed.node = {
ip.list = [
  "47.252.81.126:18888",
  "47.252.80.163:18888",
  "47.252.84.148:18888",
  "47.252.80.45:18888",
  "47.252.86.9:18888",
  "47.252.81.42:18888",
  "47.252.84.110:18888",
  "47.252.81.102:18888",
  "47.252.80.220:18888",
  "47.252.80.96:18888",
  "47.252.84.22:18888",
  "47.252.81.33:18888",
  "47.252.86.18:18888",
  "47.252.84.177:18888",
  "47.252.80.3:18888",
  "47.252.81.135:18888",
  "47.252.81.153:18888",
  "47.252.87.234:18888",
  "47.252.85.150:18888",
  "47.252.81.120:18888",
  "47.252.87.12:18888",
  "47.252.84.142:18888",
  "47.252.84.58:18888",
  "47.252.84.249:18888",
  "47.252.84.209:18888",
  "47.252.84.59:18888",
  "47.252.85.102:18888"
  ]
}
### 2. What did you expect to see?
I expect the testNetNode can connect to Shasta testnet and sync the blocks.

### 3. What did you see instead?
I found that the configuration of test_net_config.conf is not the correct node of shasta.
For example, I query the hash of 84,289 height:
000000000001494160272893c41989f1137a09942a2b68a6dfc301b36b41c082
which is different from mainNet and shastaNet.

**Comment by apa7** [comment]
My test_net_config file is from this link:
https://github.com/tronprotocol/tron-deployment/blob/master/test_net_config.conf

**Comment by jiangyy0824** [comment]
@apa7 You are right. The test_net_config is not for shasta, this is for http://testnet.tronex.io/
Local node is restricted to join Shasta.

**Comment by andkom** [comment]
@apa7 did you found config for shasta?
